src/revpi-functions

```diff
@@ -9,3 +9,11 @@ cm1-detection() {
     grep -m 1 "model name" /proc/cpuinfo | grep -q v6l
     return $?
 }
+
+# Check existance of HAT EEPROM
+has-hat-eeprom() {
+	if [ -d /proc/device-tree/hat ]; then
```

Comment:
@iluminat23 could you give the path or the link of the document again here?

Reply:
https://github.com/RevolutionPi/revpi-hat-eeprom/blob/master/RevPi-HAT-EEPROM-Format.md#custom-atom-

Reply:
It is optional to have `/proc/device-tree/hat/custom_0` ?

Reply:
Not for the RevPi-HAT-EEPROM only for a RPi HAT EEPROM